content/en/platform/corda/4.8/open-source/running-a-node-remotely.md

```diff
@@ -0,0 +1,37 @@
+---
+aliases:
+- /head/running-a-node-remotely.html
+- /HEAD/running-a-node-remotely.html
+date: '2020-04-07T12:00:00Z'
+menu:
+  corda-os-4-8:
+    identifier: corda-os-4-8-running-a-node-remotely
+    parent: corda-os-4-8-corda-nodes-index
+    weight: 1140
+tags:
+- running
+- node
+title: Running nodes remotely
+---
+
+# Running nodes remotely
+
+{{< note >}}
```

Comment:
Does this make sense? Is this whole doc just for testing and the real 'running a node remotely' doc is somewhere else? Can you figure out a way of making this make better sense?